src/components/CategoriesPills.js

```diff
@@ -81,7 +60,11 @@ class CategoriesPills extends React.PureComponent<Props> {
               onContentSizeChange={this.scrollToLastPill}
             >
               {Array.from(selectedCategories).map(name => (
-                <CategoryPill key={name} name={name} />
+                <CategoryPill
+                  key={name}
+                  name={name}
+                  style={styles.categoryPill}
```

Comment:
I'm hesitant to pass styles to a component. I have often found a better option is to wrap the component with layout components. This ties in with single responsibility, and also enables reuse of both this component and layout primitives. I'll let you decide for this PR, but I think we should move to creating layout components eventually.

Reply:
That's an interesting one. I think we agree that as long as it doesn't limit composability it won't be a problem. 
I might be biased towards keeping it like this, because I haven't seen a good example of layout components being extensively used. Not sure if this project will grow that much to have a design system where we can get to that granularity of re-use. 
I hope to come back to this when we have a few more use cases.